v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -73,4 +73,64 @@ public interface ElasticsearchWriteOptions extends PipelineOptions {
   Long getMaxRetryDuration();
 
   void setMaxRetryDuration(Long maxRetryDuration);
+
+  @Description("A property in the document being indexed whose value specifies which index to add the doc to (takes precendence over an index UDF)")
+  String getPropertyAsIndex();
+
+  void setPropertyAsIndex(String propertyAsIndex);
+
+  @Description("GCS path to JavaScript udf source for index extraction function")
```

Comment:
UDF is still spelled "udf" in a few places.

Reply:
I believe I've updated all instances to UDF, are there still any left?